scripts/build-index.py

```diff
@@ -44,7 +44,7 @@ def build_index(repo_dir):
             dir[0].title(),
             dir
         )
-    ) for dir in package_dirs]
```

Comment:
Thanks for the PR. Can we change this to be less platform specific and just exclude any hidden file? We can check if it starts with a period `.`.